Rubberduck.VBEEditor/SafeComWrappers/Abstract/IWindow.cs

```diff
@@ -5,6 +5,7 @@ namespace Rubberduck.VBEditor.SafeComWrappers.Abstract
 {
     public interface IWindow : ISafeComWrapper, IEquatable<IWindow>
     {
+        IntPtr RealHwnd { get; set; }
```

Comment:
I think we'll need some XML-doc here. The `HWnd` is the one returned from the underlying COM object (and 0 half the time); `RealHWnd` is the actual useful one, right? Would it be a good idea to simply change the `HWnd` implementation to return the "real hwnd" instead?